src/plotly.js

```diff
@@ -25,8 +25,7 @@ exports.Lib = require('./lib');
 exports.util = require('./lib/svg_text_utils');
 exports.Queue = require('./lib/queue');
 
-// plot icons svg and plot css
-exports.Icons = require('../build/ploticon');
```

Comment:
That's here: https://github.com/plotly/plotly.js/pull/23/files#diff-1fdf421c05c1140f6d71444ea2b27638R36